LICENSE.txt

```diff
@@ -609,4 +609,10 @@ Copyright 2022 Apache Lucene.NET
 
 Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.  You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0 Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the License for the specific language governing permissions and limitations under the License.
 
+Microsoft.Extensions.Configuration.Json v6.0.0
+The MIT License (MIT)
+https://github.com/dotnet/runtime/blob/main/LICENSE.TXT
 
+CsvHelper v30.0.1
+Dual licensing under MS-PL and Apache 2.0
+https://github.com/JoshClose/CsvHelper/blob/master/LICENSE.txt
```

Comment:
Please select the Apache 2.0 license text and include it in this file, see the pattern which includes copyright data the license header.